LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -1023,6 +976,57 @@ LogFileMonitor::LogFilesChangeHandler()
 }
 
 
+DWORD LogFileMonitor::GetLogFileEventStatus(DirChangeNotificationEvent& event)
+{
+    DWORD status = ERROR_SUCCESS;
+    switch (event.Action)
+    {
+        case EventAction::Add:
+        {
+            status = LogFileAddEventHandler(event);
+            break;
+        }
+
+        case EventAction::Modify:
+        {
+            status = LogFileModifyEventHandler(event);
+            break;
+        }
+
+        case EventAction::Remove:
+        {
+            status = LogFileRemoveEventHandler(event);
+            break;
+        }
+
+        case EventAction::RenameOld:

```

Comment:
Since we're not doing anything for this, could it be taken down to be right before `default:`, then they can share the body:

```cc
case EventAction::RenameOld:
default:
    // do nothing
    break;
```